src/btree/bt_split.c

```diff
@@ -1699,6 +1699,8 @@ __wt_multi_to_ref(WT_SESSION_IMPL *session, WT_PAGE *page, WT_MULTI *multi, WT_R
         WT_RET(__wt_calloc_one(session, &ref->page_las));
         *ref->page_las = multi->page_las;
         ref->page_las->birthmarks = multi->page_las.birthmarks;
+        ref->page_las->max_las_key = multi->page_las.max_las_key;
+        ref->page_las->min_las_key = multi->page_las.min_las_key;
```

Comment:
The `multi->page_las` contents are copied into `ref->page_las` including the pointers. so freeing them at `__free_page_modify()` may introduce problems?

Reply:
Yes; that's why I'm suggesting these values be cleared from the original in order to avoid any future problems, when they're assigned. Basically, we should free them when we discard the page, but we should never have more than a single reference to any of them.

Reply:
Cleared all the contents of multi-block entry once the LAS contents are copied into WT_REF.